README.md

```diff
@@ -43,32 +41,52 @@ preprocessors:
 `aliases`
 :   Mapping from aliases to Git repository URLs. Once defined here, an alias can be used to refer to the repository instead of its full URL.
 
+    >    **Note**
+    >
+    >    Aliases are available only within the legacy syntax of include statements (see below).
+
     For example, if you set this alias in the config:
 
         - includes:
-          aliases:
-            foo: https://github.com/boo/bar.git
-            baz: https://github.com/foo/far.git#develop
+            aliases:
+                foo: https://github.com/boo/bar.git
+                baz: https://github.com/foo/far.git#develop
 
-    you can include README.md file content from this repository using this syntax:
+    you can include the content of the files `doc.md` from these repositories using the following syntax:
```

Comment:
you can include the content of `doc.md` files from these repositories using the following syntax: